tests/test_cc_socket_can.py

```diff
@@ -193,27 +193,27 @@ def test_cc_close(logging_requested, mock_can_bus, mocker):
 @pytest.mark.parametrize(
     "parameters",
     [
-        (b"\x10\x36", 0x0A, True),
-        (b"\x10\x36", None, True),
-        (b"\x10\x36", 10, True),
-        (b"", 10, True),
-        (message_with_tlv, 0x0A, False),
-        (message_with_no_tlv, 0x0A, False),
-        (message_with_no_tlv,),
-        (message_with_no_tlv, 36),
+        {"msg": b"\x10\x36", "remoet_id": 0x0A, "raw": True},
```

Comment:
```suggestion
        {"msg": b"\x10\x36", "remote_id": 0x0A, "raw": True},
```
Typo to be fixed in all parameters